eth/vm/base.py

```diff
@@ -81,11 +81,11 @@
 
 
 class BaseVM(Configurable, ABC):
-    block = None  # type: BaseBlock
-    block_class = None  # type: Type[BaseBlock]
-    fork = None  # type: str
-    chaindb = None  # type: BaseChainDB
-    _state_class = None  # type: Type[BaseState]
+    block: BaseBlock = None
+    block_class: Type[BaseBlock] = None
+    fork: str = None    # noqa
```

Comment:
The `noqa` should include the type of error that we are silencing. Otherwise, this line silences all types of lint errors. What is the warning?

Reply:
E071 multiple statements on one line